index.xhtml

```diff
@@ -15,96 +15,113 @@ canonical: /
+          <img alt="Discord" src="/images/discord.svg" />
+        </a>{% endif %}
+        {% if project.matrix %}<a href="https://matrix.to/#/{{ project.matrix }}">
+          <img alt="Matrix" src="/images/matrix.svg" />
+        </a>{% endif %}
+        {% if project.libera %}<a href="ircs://irc.libera.chat:6697/{{ project.libera }}">
+          <img alt="Libera.Chat" class="nofilter" src="/images/libera.svg" />
+        </a>{% endif %}
+        {% if project.discourse %}<a href="{{ project.discourse }}">
+          <img alt="Discourse" src="/images/discourse.svg" />
+        </a>{% endif %}
+        {% if project.reddit %}<a href="https://reddit.com/r/{{ project.reddit }}">
+          <img alt="Reddit" src="/images/reddit.svg" />
+        </a>{% endif %}
+        {% if project.github %}<a href="https://github.com/{{ project.github }}">
+          <img alt="GitHub" src="/images/github.svg" />
+        </a>{% endif %}
+        {% if project.gitlab %}<a href="https://gitlab.com/{{ project.gitlab }}">
+          <img alt="GitLab" src="/images/gitlab.svg" />
+        </a>{% endif %}
+      </span>
+      <span class="author">
+        {% if project.organization %}
         {% if project.organization.website %}<a href="{{ project.organization.website }}">{% endif %}
-          {{ project.organization.name | default: project.organization }}<!--
-        {% if project.organization.website %}--></a><!--{% endif %}
-        {% if project.authors %}-->, including <!--{% endif %}
-        {% if project.author %}-->, including <!--{% endif %}-->
-      {% endif %}
-      {% assign num_authors = project.authors | size %}
-      <!-- {% if num_authors < 2 %}-->
+          {{ project.organization.name | default: project.organization }}
+          <!--
+        {% if project.organization.website %}-->
+        </a>
+        <!--{% endif %}
+        {% if project.authors %}-->, including
+        <!--{% endif %}
+        {% if project.author %}-->, including
+        <!--{% endif %}-->
+        {% endif %}
+        {% assign num_authors = project.authors | size %}
+        <!-- {% if num_authors < 2 %}-->
         {% assign author = project.authors[0] | default: project.author %}
         {% if author.website %}<a href="{{ author.website }}">{% endif %}
           {{ author.name | default: author }}
-        {% if author.website %}</a>{% endif %}
-      {% else %}-->
+          {% if author.website %}</a>{% endif %}
+        {% else %}-->
         {% assign authors_except_last = project.authors | reverse | shift | reverse %}
         {% for author in authors_except_last %}
-          {% if author.website %}<a href="{{ author.website }}">{% endif %}
-            {{ author.name | default: author }}<!--
-          -->{% if author.website %}</a>{% endif %},
+        {% if author.website %}<a href="{{ author.website }}">{% endif %}
+          {{ author.name | default: author }}
+          <!--
+          -->{% if author.website %}
+        </a>{% endif %},
```

Comment:
These weird comments were like that for a reason. HTML folds multiple spaces to one, but it does not remove spaces between tags entirely. This creates a bunch of extraneous spaces which break the site's formatting (e.g. `many , including Paul Bone`. Please revert or correct this portion of the change.